apps/api_web/lib/api_web/params.ex

```diff
@@ -283,7 +283,7 @@ defmodule ApiWeb.Params do
           |> String.split(",", trim: true)
           |> Enum.map(&(&1 |> String.split(".") |> List.first()))
 
-        bad_includes = split -- includes
+        bad_includes = Enum.filter(split, fn el -> el not in includes end)
```

Comment:
This is effectively quadratic, as it's iterating over two linked lists. Could you convert `includes` to a `MapSet`?